clusters/resource_cluster.go

```diff
@@ -47,9 +48,19 @@ func SparkConfDiffSuppressFunc(k, old, new string, d *schema.ResourceData) bool
 	return false
 }
 
+func AwsAttribsDiffSuppressFunc(k, old, new string, d *schema.ResourceData) bool {
+	if strings.HasPrefix(k, "aws_attributes.") &&
```

Comment:
Just check for key as a string - I've made GoLang custom struct fields to translate as always-single-element-lists, so you can be sure of first element being zero

Reply:
done